gomock/call.go

```diff
@@ -21,6 +21,12 @@ import (
 	"strings"
 )
 
+type WrapperCall interface {
```

Comment:
Why is this called WrapperCall, what does it mean? What purpose does this achieve?